CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 7.4.1 - 2019/10/22
```

Comment:
Bump to 23

Reply:
fixed in https://github.com/elastic/ansible-beats/pull/49/commits/328c2f98a893d24dceaac80500c72f12c016c6d4